### PR TITLE
Committing changes to master readme

### PR DESCRIPTION
**Pull request by harinikmsft**
This PR tracks changes to master readme to reflect the vnext effort and other changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
+
+![Hero Image with Logo](./.github/hero2.png)
 
 [React Native](http://facebook.github.io/react-native) enables you to build world-class application experiences on native platforms using a consistent developer experience based on JavaScript and [React](http://facebook.github.io/react). The focus of React Native is on developer efficiency across all the platforms you care about - learn once, write anywhere. Facebook uses React Native in multiple production apps and will continue investing in React Native.
 
-This repository adds support for the [Windows 10 SDK](https://developer.microsoft.com/en-us/windows/downloads), which allows you to build apps for:
-* Windows 10
-* Windows 10 Mobile
-* [Xbox One (UWP)](https://msdn.microsoft.com/en-us/windows/uwp/xbox-apps/index)
+This repository adds support for the [Windows 10 SDK](https://developer.microsoft.com/en-us/windows/downloads), which allows you to build apps for [all devices supported by Windows 10](https://developer.microsoft.com/en-us/windows/get-started-windows-10) including PCs, tablets, 2-in-1s, Xbox, Mixed reality devices etc.,
```
Comment (kmelmon) [inline README.md]: >, [](start = 320, length = 1)

nit:  remove ','

File: README.md [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
+We are in the process of refactoring the native bridge implementation for `react-native-windows` to better align with upcoming changes related to the [React Native Fabric Architecture](https://github.com/react-native-community/discussions-and-proposals/issues/4). This effort is currently taking place in the [vnext](https://github.com/Microsoft/react-native-windows/tree/master/vnext) subdirectory within this GitHub repo. You can read more about this effort [here](vnext/README.md). We are tracking progress on this work through [Projects](https://github.com/Microsoft/react-native-windows/projects) and [Issues](https://github.com/Microsoft/react-native-windows/issues?q=is%3Aopen+is%3Aissue+label%3Arnwcpp) on this GitHub repo.
```
Comment (kmelmon) [inline README.md]: >refactoring the native bridge implementation for `react-native-windows` to better align with upcoming changes related to the [](start = 25, length = 124)

I've been wondering about this statement for a while.  In my mind this is pretty clearly not the reason for a vnext.  Recalling a recent email thread discussion, I'm also not sure we should make such a strong claim about aligning with Fabric (at least not yet).

Can I suggest something like:
We are in the process of re-implementing react-native-windows in C++, for better performance, and to better align with the shared C++ react-native core as it evolves.

Note that I don't feel super strongly about not mentioning Fabric but the above seems like a more true statement to me.

File: README.md (hunk 1 above)
Comment (harinikmsft) [reply README.md]: Addressed with [new commit](https://github.com/Microsoft/react-native-windows/pull/2409/commits/14cf06fdfa0fabbc7599a0234028bbf79e5b144b)

File: README.md (hunk 2 above)
Comment (harinikmsft) [reply README.md]: Addressed with [new commit](https://github.com/Microsoft/react-native-windows/pull/2409/commits/14cf06fdfa0fabbc7599a0234028bbf79e5b144b)

**Pull request merged**